packages/squiggle-lang/src/rescript/FR/FR_Dict.res

```diff
@@ -170,4 +204,24 @@ let library = [
     ],
     (),
   ),
+  Function.make(
+    ~name="map",
+    ~nameSpace,
+    ~requiresNamespace=true,
+    ~output=EvtRecord,
+    ~examples=[`Dict.map({a: 1, b: 2}, {|x| x + 1})`],
+    ~definitions=[
+      FnDefinition.make(
+        ~name="map",
+        ~inputs=[FRTypeDict(FRTypeAny), FRTypeLambda],
+        ~run=(inputs, env, reducer) =>
```

Comment:
There's some inconsistency about this in the FR (my fault due to recent interpreter changes), but the second parameter to `run` is currently `Reducer_T.context`, not `Reducer_T.environment`, so that arg should be called `context` to avoid the confusion.

Reply:
Sure. I'll make a PR that fixes the old uses of `env` or `environment` then.